setup.py

```diff
@@ -126,6 +126,9 @@
         'invenio_base.apps': [
             'invenio_oauthclient = invenio_oauthclient:InvenioOAuthClient',
         ],
+        'invenio_base.api_apps': [
```

Comment:
I don't see any quick solution for it right now. However, we should create a separate class `InvenioOAuthClientREST` which will not do https://github.com/inveniosoftware/invenio-oauthclient/blob/master/invenio_oauthclient/ext.py#L145-L158